src/main/java/pl/byczazagroda/trackexpensesappbackend/dto/UserDTO.java

```diff
@@ -0,0 +1,8 @@
+package pl.byczazagroda.trackexpensesappbackend.dto;
+
+import pl.byczazagroda.trackexpensesappbackend.model.UserStatus;
+
+public record UserDTO (
+        Long id, String user_name,  String email,
```

Comment:
We got here `userStatus` but `user_name`?
Only one space character between `user_name,` and `String email`.
One space character between ')' and '{'is needed as well.
The maximum line length is 120 characters, not just 80.

```suggestion
public record UserDTO (Long id, String userName, String email, String password, UserStatus userStatus) {}
```